WTWSMS/history/characters/kannada.txt

```diff
@@ -111,7 +111,7 @@
 }
 
 74404 = {
-	name="Flavius Julius" 
+	name="Majorian"
```

Comment:
Why did you rename him? Flavius Julius Valerius Majorianus would work, but just Majorian poses a problem in French and Spanish.

Reply:
He isn't known as "Flavius Julius". I will change it to his simple Latin name, Maiorianus.

Reply:
Okay.